shell/recover.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/expect
+
+set password [lindex $argv 0]
+set walletbinary [lindex $argv 1]
+set network [lindex $argv 2]
+set passphrase [lindex $argv 3]
+
+	cd $walletbinary
```

Comment:
Why these empty spaces?

Reply:
This code is not even used anymore, I can do everything with the expect of rust itself with that library. I will delete this file